ios/engine/KMEI/KeymanEngine/Classes/Migrations.swift

```diff
@@ -364,6 +374,7 @@ public enum Migrations {
     return Font(family: family, source: files)
   }
 
+  // OLD legacy migration.
```

Comment:
Will the FV app run this migration going from 13.0 to 14.0?
The app has a lot of keyboards we prob don't want to copy to separate folders since it's going to be installing fv_all.kmp into a single folder.

Reply:
The FV app apparently doesn't even _do_ migrations.

For a proper 14.0 FV, we'll want to install from `fv_all.kmp` anyway, as you've noticed.  As it's very eager to preload resources, its resource handling pattern would probably still negate the need for migrations within the FV app.

Reply:
We shouldn't need a migration for the keyboards for the FV app -- just remember the keyboards the user has chosen to use.